tests/generator/test_bonds.py

```diff
@@ -197,7 +197,82 @@ def test_bond_with_parameters(self):
                                             'DownDelaySec=30ms\n'
                                             'FailOverMACPolicy=none\n'
                                             'GratuitousARP=10\n'
-                                            'PacketsPerSlave=10\n'
+                                            'PacketsPerSlave=10\n'  # wokeignore:rule=slave
+                                            'PrimaryReselectPolicy=none\n'
+                                            'ResendIGMP=10\n'
+                                            'LearnPacketIntervalSec=10\n',
+                              'bn0.network': '''[Match]
+Name=bn0
+
+[Network]
+DHCP=ipv4
+LinkLocalAddressing=ipv6
+ConfigureWithoutCarrier=yes
+
+[DHCP]
+RouteMetric=100
+UseMTU=true
+''',
+                              'eno1.network': '[Match]\nName=eno1\n\n'
+                                              '[Network]\nLinkLocalAddressing=no\nBond=bn0\n',
+                              'switchports.network': '[Match]\nDriver=yayroute\n\n'
+                                                     '[Network]\nLinkLocalAddressing=no\nBond=bn0\n'})
+
+    def test_bond_with_parameters(self):
+        self.generate('''network:
+  version: 2
+  ethernets:
+    eno1: {}
+    switchports:
+      match:
+        driver: yayroute
+  bonds:
+    bn0:
+      parameters:
+        mode: 802.3ad
+        lacp-rate: 10
+        mii-monitor-interval: 10
+        min-links: 10
+        up-delay: 20
+        down-delay: 30
+        all-slaves-active: true  # wokeignore:rule=slave
+        transmit-hash-policy: none
+        ad-select: none
+        arp-interval: 15
+        arp-validate: all
+        arp-all-targets: all
+        fail-over-mac-policy: none
+        gratuitious-arp: 10
+        packets-per-slave: 10  # wokeignore:rule=slave
```

Comment:
Should we also simply use `packets-per-member` here as well?

Reply:
These instances were left there on purpose. As these are valid configuration we can't remove just yet, we should have tests making use of them.